src/components/shared/links.js

```diff
@@ -44,6 +44,7 @@ function ListLink({ title, url }) {
 function Links({ links = [], title, headingLevel, description }) {
   const isGrid = links?.some((item) => item.image);
   const Heading = getHeadingLevel(headingLevel);
+  const numLinks = links.length;
   // const LinkHeading = title ? getHeadingLevel(headingLevel, 1) : "p";
```

Comment:
You could delete this commented-out line, but aside from that, it's good to go.

Reply:
Good catch - removed 👍